pyinfra/facts/util/distro.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, unicode_literals
+
+import os
+
+import distro
+
+
+def get_distro_info(root_dir):
+    # We point _UNIXCONFDIR to root_dir
+    old_value = distro._UNIXCONFDIR
```

Comment:
This part is bit hacky but it does the trick. Master version of distro has support for passing a root_dir to LinuxDistribution object __init__. Once that's release and this is accepted we can make use of it instead of resorting to this.

Reply:
👍 works for now!